projects/storefrontlib/src/lib/store-finder/store/actions/find-stores.action.ts

```diff
@@ -26,11 +28,17 @@ export class FindStores implements Action {
     public payload: {
       queryText: string;
       longitudeLatitude?: LongitudeLatitude;
+      useMyLocation?: boolean;
       searchConfig?: SearchConfig;
     }
   ) {}
 }
 
+export class FindStoresWithMyLocation implements Action {
```

Comment:
I don't think we need a new action. We still can you use the old  FindStoresAction with the geodata gathered beforehand.